tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -327,6 +329,160 @@ def is_ssd(rotational):
     return False
 
 
+def get_node_network():
+    """return
+
+           {"interface_name": {"ipv4": ["ipv4address", ...],
+                               "ipv6": ["ipv6address, ..."],
+                               "netmask": ["subnet", ...],
+                               "subnet": "subnet",
+                               "status":"up/down",
+                               "interface_id": "",
+                               "sysfs_id": "",
+                               "device_link": "",
+                               "interface_type": "",
+                               "model": "",
+                               "driver_modules": "",
+                               "drive": "",
+                               "hw_address": "",
+                               "link_detected": ""
+                              }, ...
+           }
+    """
+    rv = {}
+    network_interfaces = get_node_interface()
+    cmd = Command({"_raw_params": 'hwinfo --network'})
+    out, err = cmd.start()
+    if not err:
+        if not out['stderr']:
+            out = out['stdout']
+            for interface in out.split('\n\n'):
+                devlist = {"interface_id": "",
+                           "sysfs_id": "",
+                           "device_link": "",
+                           "interface_type": "",
+                           "model": "",
+                           "driver_modules": "",
+                           "drive": "",
+                           "interface_name": "",
+                           "hw_address": "",
+                           "link_detected": ""}
+                for line in interface.split('\n'):
```

Comment:
The below section, seems a bit overkill, If any one have any better option please comment